Patch to fix the redefinition of an index variable caught by GCC 11.
author: Micael Oliveira (Max Planck Institute for the Structure and Dynamics of Matter)
diff -rup futile-1.8.3.orig/flib/f_input_file.f90 futile-1.8.3/flib/f_input_file.f90
--- futile-1.8.3.orig/flib/f_input_file.f90     2018-12-13 10:27:01.000000000 +0100
+++ futile-1.8.3/flib/f_input_file.f90          2022-01-07 11:35:42.036739693 +0100
@@ -172,7 +172,6 @@ contains
     type(dictionary), pointer :: dict
     character(len = *), intent(in) :: file, key
 
-    integer :: i
     type(dictionary), pointer :: ref,iter
     character(len = max_field_length) :: val, profile_
     double precision, dimension(2) :: rg
@@ -366,6 +365,7 @@ contains
       character(max_field_length), dimension(:), allocatable :: keys
       double precision :: var
       integer :: dlen, skeys
+      integer :: i
 
       if (associated(dict%child)) then
          if (dict_len(dict) >= 1) then
